src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int num;

```

Comment:
Use the short form of words for class variables naming - bad practice
```suggestion
    private int number;
```